bin/parse

```diff
@@ -14,7 +15,47 @@ switch (args.paradigm) {
     // load and use object-oriented parser
     break
   case 'functional':
-    // load and use functional parser
+    console.log(
```

Comment:
You can use a super-parser function here, to keep it clean and simple, while keeping your chain of functions hidden 'under the hood':
```js
    const result = functional.parseMarkdown( fs.readFileSync(fileName, 'utf8') )
    console.log('functional', result);
```